bakery/common_test.go

```diff
@@ -0,0 +1,149 @@
+package bakery_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"golang.org/x/net/context"
+	gc "gopkg.in/check.v1"
+	"gopkg.in/macaroon.v2-unstable"
+
+	"gopkg.in/macaroon-bakery.v2-unstable/bakery"
+	"gopkg.in/macaroon-bakery.v2-unstable/bakery/checkers"
+)
+
+// BC holds the testing background context - its associated time when checking
+// time-before caveats will always be the value of epoch.
+var BC = checkers.ContextWithClock(context.Background(), stoppedClock{epoch})
```

Comment:
Since it's in a test, it doesn't matter and I on balance prefer the visual distinctiveness of BC vs bc.
I think I'm swaying away from BC towards testContext actually. I'll change everything to use that.